src/gitUtils.ts

```diff
@@ -9,7 +9,7 @@ export const setupUser = async () => {
   await exec("git", [
     "config",
     "user.email",
-    `"github-actions[bot]@users.noreply.github.com"`,
+    `"41898282+github-actions[bot]@users.noreply.github.com"`,
```

Comment:
is this number stable? 🤔

Reply:
I guess it's its ID... so it should be stable 🤷‍♂️

Reply:
Yeah I think it should. The earliest mention [I found](https://github.com/orgs/community/discussions/26560#discussioncomment-3252340) was from 2020, and the checkout action also [mentions it](https://github.com/actions/checkout?tab=readme-ov-file#push-a-commit-using-the-built-in-token). So likely safe enough.